jmason87/ygs-watch-fe
# sets page and route restructuring

## What does this PR do?
this PR adds a sets index and store view, routes defined, store setup for create and read. Add sets page will be used for admin features in the future
## Related PRs

## Testing Steps
- pull branch
- click the hockey link at top of page
- see that it takes you './hockey/sets'
- this page  should be a list of buttons of years
- clicking a button should take you to './hockey/sets/{the year you clicked on}'
- this page right now is just a table of players
## Checklist before merging
